src/App.js

```diff
@@ -1,14 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import LikeCounter from './components/LikeCounter'
 
 const App = () => {
+  const logCopy = chatMessages.map((messages)=> {return {...messages}})
+  const [messages,changeMessages] = useState(logCopy)
+  const [likes,updateLikes] = useState(0)
+
+
+  const countLikes = (mg) => {
+    let likeCount = 0
+    for (const msg of mg) {
+      if (msg.liked===true){
+        likeCount++
+    }
```

Comment:
Since `msg.liked` is a boolean, we don't have to `===true` here! In fact, we can shorten these lines to:
```
msg.liked && likeCount++
```